src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -417,29 +413,29 @@ private List<OpenApiParameter> GetOpenApiParameters(MethodInfo methodInfo, Endpo
         return openApiParameterContent;
     }
 
-    private (bool isBodyOrForm, ParameterLocation? locatedIn) GetOpenApiParameterLocation(ParameterInfo parameter, RoutePattern pattern, bool disableInferredBody)
+    private (bool isBodyOrForm, ParameterLocation? locatedIn, bool isParameter) GetOpenApiParameterLocation(ParameterInfo parameter, RoutePattern pattern, bool disableInferredBody)
```

Comment:
Is it necessary to add `isParameter` to the tuple? It seems like it's always equivalent to `locatedIn is not null`.